src/localmodeling.jl

```diff
@@ -168,37 +212,43 @@ end
 
 
 """
-`TSP(tree,R,p,LocalModel,method,f)`
+    TSP(tree,R,q=R[end],p,LocalModel,method,f) -> s_pred
 
-This function is a simple tool for Time Series Prediction.
+A simple tool for Time Series Prediction.
```

Comment:
Don't capitalize unless it's a name. also, throughout the library I use "timeseries" without space between. Let's try to keep the convention.

Reply:
In addition, the description is not sufficient. The first part of the docstring should say what each argument is, unless they are keyword arguments. 

I do not agree that the highest level function should take as an argument a Tree. You gain nothing from this argument being given. Just create it inside the function, since the user already gives you a reconstruction.

Is there a reason to ask for the `tree` which I do not see?